Classes/UT3Nightshade.uc

```diff
@@ -156,7 +174,7 @@ function bool NoObstacle()
 	else
 		return false;
 }
-/* � Type:
+/* � Type:

```

Comment:
This character got messed up too.

Reply:
That's weird, didn't even know this character was special